plugins/hotfix.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: hotfix
+spec:
+  version: "v0.0.3"
+  shortDescription: dumb file transfer to your labelled pods
+  homepage: https://steady.supply/git/kubectl-hotfix
+  description: |
```

Comment:
This description reads more like a man page.

Please focus on what your plugin does and why a user would need it. The details you give here should be moved to the commands `--help` output.